ospfd/ospf_lsa.h

```diff
@@ -338,6 +338,7 @@ extern struct ospf_lsa *ospf_translated_nssa_refresh(struct ospf *,
 						     struct ospf_lsa *,
 						     struct ospf_lsa *);
 extern struct ospf_lsa *ospf_translated_nssa_originate(struct ospf *,
+						       struct ospf_lsa *,
```

Comment:
Please name the variables.  I know the file does not have it, but fix the function you modify please

Reply:
okay, done